platform/posix/transport/CMakeLists.txt

```
@@ -40,12 +40,47 @@ target_link_libraries( openssl_posix
                           # requires explicit linking.
                           ${CMAKE_DL_LIBS} )
 
+# Set path to corePKCS11 and it's third party libraries.
+set(COREPKCS11_LOCATION "${CMAKE_SOURCE_DIR}/libraries/standard/corePKCS11")
+set(CORE_PKCS11_3RDPARTY_LOCATION "${COREPKCS11_LOCATION}/source/dependency/3rdparty")
+
+# Include PKCS #11 library's source and header path variables.
+include( ${COREPKCS11_LOCATION}/pkcsFilePaths.cmake )
+
+list(APPEND PKCS_SOURCES
+    "${COREPKCS11_LOCATION}/source/portable/posix/core_pkcs11_pal.c"
+    "${CORE_PKCS11_3RDPARTY_LOCATION}/mbedtls_utils/mbedtls_utils.c"
+)
+
+# Create target for POSIX implementation of MbedTLS.
+add_library( mbedtls_pkcs11_posix
```

I think it is valuable to mention the `transport_` prefix across all the transport implementation target libraries
```suggestion
add_library( transport_mbedtls_pkcs11_posix
```